### ISSUE TITLE
php tags?

### ISSUE THREAD

**Issue by smlombardi** [simurai/duotone-dark-syntax#15]
I sometimes work with php.  It would be nice for some php, wordpress love (maybe i will do a pull request) but I would say, at the very least the php TAGS `<?php ... ?>` as shown below should be more important than comments.  Bright orange?  Purple?

![php](https://cloud.githubusercontent.com/assets/835148/13128054/06a72484-d5a1-11e5-99e6-2d269019a5fb.png)

**Comment by simurai** [comment]
Yes, they feel like part of the comment. Maybe it could be `.uno-1();`, same as [html elements](https://github.com/simurai/duotone-dark-syntax/blob/master/styles/languages/_base.less#L34-L39)?
